src/main/java/com/minecolonies/structures/helpers/Structure.java

```diff
@@ -329,14 +329,14 @@ private void renderGhost(final World world, final ModelHolder holder, final Enti
                     this.mc.getTextureManager().bindTexture(TextureMap.LOCATION_BLOCKS_TEXTURE);
                     ForgeHooksClient.setRenderLayer(layer);
                     this.renderGhostBlock(world, holder, player, layer, existingModel, partialTicks);
-                    holder.rendered = true;
+                    holder.setRendered(true);;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)